pkg/providers/aws/provider_postgres.go

```diff
@@ -326,22 +342,24 @@ func (p *PostgresProvider) reconcileRDSInstance(ctx context.Context, cr *v1alpha
 			return nil, croType.StatusMessage(fmt.Sprintf("reconcileRDSInstance() in progress, current aws rds resource status is %s", *foundInstance.DBInstanceStatus)), nil
 		}
 
-		// check if found instance and user strategy differs, and modify instance
-		logger.Infof("found existing rds instance: %s", *foundInstance.DBInstanceIdentifier)
-		mi, err := buildRDSUpdateStrategy(rdsCfg, foundInstance, cr)
-		if err != nil {
-			errMsg := fmt.Sprintf("error building update config for rds instance: %s", *foundInstance.DBInstanceIdentifier)
-			return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
-		}
-		if mi != nil {
-			_, err := rdsSvc.ModifyDBInstance(mi)
+		if maintenanceWindow {
```

Comment:
this part of code seems not covered by Unit tests

Reply:
I think this is covered by the new / modified original tests in ther postgres_provider_test.go. You can see the covered lines here https://app.codecov.io/gh/integr8ly/cloud-resource-operator/pull/468 and all of the lines in that maintenanceWindow block are covered.